### PR TITLE
fix(shell-autocompletion.md): fix docs

### PR DESCRIPTION
**Pull request by wiasliaw**
#261

### CODE REVIEW

File: src/config/shell-autocompletion.md [hunk 1]
```diff
@@ -17,6 +17,13 @@ cast completions zsh > $HOME/.oh-my-zsh/completions/_cast
 source $HOME/.zshrc
 ```
 
+Since [zsh v5.0.7](https://zsh.sourceforge.io/releases.html), where to place the completions scripts had been change.
+You just need to place `_forge` and `_cast` into `/usr/local/share/zsh/site-functions/`:
+```sh
+forge completions zsh > /usr/local/share/zsh/site-functions/_forge
+cast completions zsh > /usr/local/share/zsh/site-functions/_cast
+```
```
Comment (onbjerg) [inline src/config/shell-autocompletion.md:25]: Can you replace the entire section with just like:

    ### zsh
    
    First, ensure that the following is present somewhere in your `~/.zshrc` file (if not, add it):
    
    ```sh
    autoload -U compinit
    compinit -i
    ```

    Then run:
    ```
    forge completions zsh > /usr/local/share/zsh/site-functions/_forge
    cast completions zsh > /usr/local/share/zsh/site-functions/_cast
    ```

**Review by onbjerg — CHANGES_REQUESTED** [review]

File: src/config/shell-autocompletion.md (hunk 1 above)
Comment (wiasliaw) [reply src/config/shell-autocompletion.md]: Got it.

File: src/config/shell-autocompletion.md [hunk 2]
```diff
@@ -5,16 +5,17 @@ You can generate autocompletion shell scripts for `bash`, `elvish`, `fish`, `pow
 ### zsh (`oh-my-zsh`)
```
Comment (onbjerg) [inline src/config/shell-autocompletion.md:5]:
```suggestion
### zsh
```

**Review by onbjerg — APPROVED** [review]
LGTM :smile: Thanks!

**Comment by onbjerg** [comment]
@all-contributors add @wiasliaw for maintenance

**Pull request merged**